.github/workflows/windows.yml

```diff
@@ -20,13 +21,13 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Install packages
         uses: msys2/setup-msys2@v2
         with:
           update: true
-          install: autoconf automake libtool make mingw-w64-x86_64-db mingw-w64-x86_64-gmp libdb-devel mingw-w64-x86_64-gcc flex bison gmp-devel help2man texinfo
+          install: autoconf automake libtool make mingw-w64-x86_64-db mingw-w64-x86_64-gmp libdb-devel mingw-w64-x86_64-gcc flex bison gmp-devel help2man texinfo gettext-devel
```

Comment:
I suggest to check the official MSYS2 package for dependencies

https://github.com/msys2/MINGW-packages/blob/86ee1d822d3ac1a72d011f48c045ff0a48432caf/mingw-w64-gnucobol/PKGBUILD#L20-L26

* instead of `autoconf automake libtool make`: autotools
* likely possible to remove: `gmp-devel` (this should be MSYS related, but you want to build for mingw)
* instead of `mingw-w64-x86_64-gcc`: `mingw-w64-x86_64-cc` (it is a meta package installing the former, but that was an explicit change to later support different environments than mingw-w64-x86_64 by the MSYS2 maintainers) 
* additional: `mingw-w64-x86_64-libxml2`
* additional: either `mingw-w64-x86_64-cjson` or `mingw-w64-x86_64-json-c`
* additional: either `mingw-w64-x86_64-ncurses` or `mingw-w64-x86_64-pdcurses`
* additional: perl (to run the testsuite)
* in general: switch the `w64-x86_64` to a build matrix, for example:
```
  strategy:
    matrix:
      include:
        - { sys: mingw64, env: x86_64 }
        - { sys: mingw32, env: i686 }
        - { sys: clang64, env: clang-x86_64 }
  steps:
      - name: Install packages
        uses: msys2/setup-msys2@v2
        with:
          msystem: ${{matrix.sys}}
          update: true
          install: autotools flex bison mingw-${{matrix.env}}-cc help2man texinfo perl mingw-${{matrix.env}}-gmp mingw-${{matrix.env}}-db mingw-${{matrix.env}}-cjson mingw-${{matrix.env}}-libxml2 mingw-${{matrix.env}}-ncurses
```

Reply:
Ah, this was the place where I've checked the build matrix the last time, still recommend that, including additional ucrt to help with portability tests, which get in handy for testing the inttypes / stdtypes PR #116